physics/sgscloud_radpre.F90

```diff
@@ -328,12 +331,13 @@ subroutine sgscloud_radpre_run(    &
                    else                           ! scaling function (CB2005)
                       f = 1.0
                    endif
-                   sigq = 1.5E-3 * ud_mf(i,k)/dt * f
+                   sigq = ud_mf_timeave(i,k)/dt * f
+                   !sigq = 1.5E-3 * ud_mf(i,k)/dt * f
```

Comment:
This is a bit shocking. Are you sure sigq will be ~_O_(1e-4) without this constant? If sigq is too big, you might always get cloud fractions of around 50%,

Reply:
I can check. I have output arrays of just GF cloud fraction. Where there are clouds, cloud fraction values tend to be between 30-70%.